packages/cli/src/models/ProjectApp.js

```diff
@@ -8,7 +8,8 @@ import ControllerFor from '../models/network/ControllerFor';
  */
 export default async function testApp(txParams = {}, networkFile = undefined) {
```

Comment:
I'd rename it to TestProject instead of ProjectApp, and consider supporting libs as well as apps